src/BuiltInTools/BrowserRefresh/WebSocketScriptInjection.js

```diff
@@ -69,7 +76,7 @@ setTimeout(async function () {
       document.querySelector(`link[href^="${document.baseURI}${path}"]`);
 
     // Receive a Clear-site-data header.
-    await fetch('/_framework/clear-browser-cache');
+    await fetch('_framework/clear-browser-cache');
```

Comment:
This is to make it consistent with how we fetch `_framework/blazor-hotreload` and support non-root-hosting. Please let me know if there was some specific reason why this particular URL should always be host-rooted.

Reply:
Wouldn't this bork if your page is being served from `/SubDir/MyPage`? The middleware is only listening to `$host/_framework/clear-browser-cache`?

Reply:
In that case yes, we need the leading slash. How come we don’t also need it for _framework/blazor-hotreload? I was trying to make them consistent.

Reply:
That sounds like a bug. Could you update the `_framework/blazor-hotreload` to have the leading slash?

Reply:
Done